### ISSUE TITLE
Layout issue of footer on small devices.

### ISSUE THREAD

**Issue by try-catch-stack** [COPS-IITBHU/hackalog-frontend#128]
Footer is not responsive enough and squeezes the content on small devices which may look really odd.
The layout can be made better to look good even on smaller devices.
Attaching the screenshot of the same.
 
![Screenshot from 2021-10-05 22-01-31](https://user-images.githubusercontent.com/77742477/136065608-0fb5a121-4820-4a03-ab92-eaa6e6d37607.png)

**Comment by shubhanshu02** [comment]
Yes, right! The footer needs to be redesigned in a mobile-first way.

**Comment by try-catch-stack** [comment]
> Yes, right! The footer needs to be redesigned in a mobile-first way.

Okay then , I'll change layout of the footer. Do you suggest any other changes or additions to it other than the layout?

**Comment by try-catch-stack** [comment]
> Maybe like displaying more page routes along with the contributors' page. I guess the following links can be listed in the footer thinking:
> 
>     * Home - `/`
> 
>     * Contributors Page - `/contributors`
> 
>     * Hackathon List page - `/hackathons`

Sure, I would like to work on it.

**Comment by shubhanshu02** [comment]
Great, I have assigned this issue to you.

**Comment by try-catch-stack** [comment]
How about adding a contact us or an email field in the footer so that we can get back to the user in case of any query.

**Comment by tiger-yash** [comment]
> @tiger-yash @ankur12-1610 @Tarun-Arora @subodhk01 @m-e-l-u-h-a-n @shukapurv @soumyo123-prog @Prakhar-Agarwal-byte @shubhangi013 @garimavs Any suggestions? Or anyone else from the team?

Yes the expansion of the footer to accommodate these elements properly would be good.
@try-catch-stack just make sure that in adding the new elements you arrange them into proper columns/rows, and maybe rest we can analyse along with the PR preview.

**Comment by shubhanshu02** [comment]
> How about adding a contact us or an email field in the footer so that we can get back to the user in case of any query.

One COPS main website link also 😅 (`https://www.copsiitbhu.co.in/`). You can take the contact details from there.
